src/test/java/org/symphonyoss/symphony/messageml/elements/CodeLanguageTest.java

```diff
@@ -0,0 +1,97 @@
+package org.symphonyoss.symphony.messageml.elements;
+
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.symphonyoss.symphony.messageml.MessageMLContext;
+import org.symphonyoss.symphony.messageml.exceptions.InvalidInputException;
+import org.symphonyoss.symphony.messageml.util.TestDataProvider;
+
+@RunWith(value = Parameterized.class)
+public class CodeLanguageTest {
+
+  @Parameterized.Parameter
```

Comment:
the downside is that we are running the same test multiple times (for the ones not using the parameter)
maybe we should consider migrating to junit5?